plugin/storage/es/factory_test.go

```diff
@@ -164,11 +163,16 @@ func TestTagKeysAsFields(t *testing.T) {
 }
 
 func TestFactory_LoadMapping(t *testing.T) {
-	spanMapping5, serviceMapping5 := GetSpanServiceMappings(10, 0, 5)
-	spanMapping6, serviceMapping6 := GetSpanServiceMappings(10, 0, 6)
-	spanMapping7, serviceMapping7 := GetSpanServiceMappings(10, 0, 7)
-	dependenciesMapping6 := GetDependenciesMappings(10, 0, 6)
-	dependenciesMapping7 := GetDependenciesMappings(10, 0, 7)
+	spanMapping5, serviceMapping5, e5 := GetSpanServiceMappings(10, 0, 5, "", false)
```

Comment:
Please use `err` and `assert.NoError` immediately after like so:
```
spanMapping5, serviceMapping5, err := GetSpanServiceMappings(10, 0, 5, "", false)
assert.NoError(t, err)
spanMapping6, serviceMapping6, err := GetSpanServiceMappings(10, 0, 6, "", false)
assert.NoError(t, err)
...
```

Reply:
Yep. Fixing all of them. Could not find time last week, will work on these this week.